libcni/api.go

```diff
@@ -580,7 +580,7 @@ func (c *CNIConfig) ValidateNetworkList(ctx context.Context, list *NetworkConfig
 // It uses the same logic as ValidateNetworkList)
 // Returns a list of capabilities
 func (c *CNIConfig) ValidateNetwork(ctx context.Context, net *NetworkConfig) ([]string, error) {
-	caps := []string{}
+	var caps []string
```

Comment:
@mars1024 seems like an unrelated change?

Reply:
yeap, maybe some auto typo fix ~ I will revert it.

Reply:
Updated, thanks @dcbw